### PR TITLE
Add ESLint

### PR DESCRIPTION
**Pull request by hengfeiyang**

### CODE REVIEW

File: web/src/layouts/MainLayout.vue [hunk 1]
```diff
@@ -137,39 +143,38 @@ export default {
...
 
     const local = ref(getLocale());
-    const selectedLanguage = ref(langList.find((l) =>
-      l.value == local.value
-    ));
+    const selectedLanguage = ref(langList.find((l) => l.value == local.value));
 
-    if (!selectedLanguage || !selectedLanguage.value) {
-      selectedLanguage.value = langList[0]
+    if (!selectedLanguage.value || !selectedLanguage.value) {
```
Comment (oasisk) [inline web/src/layouts/MainLayout.vue:176]: Duplicate check for !selectedLanguage.value

File: web/src/locales/en.ts [hunk 2]
```diff
@@ -1,58 +1,58 @@
...
-      search: "Search Index",
-    },
-    template: {
-      header: "Templates",
-      add: "Add Template",
-      search: "Search Template",
-    }
-  };
+  search: {
+    selectIndex: "Select Index First",
```
Comment (oasisk) [inline web/src/locales/en.ts:3]: can we use ""Select index first"

**Review by oasisk — APPROVED** [review]

File: web/package.json [hunk 3]
```diff
@@ -29,13 +30,21 @@
   },
   "devDependencies": {
     "@quasar/vite-plugin": "^1.0.10",
+    "@typescript-eslint/eslint-plugin": "^5.14.0",
```
Comment (oasisk) [inline web/package.json:33]: had to add "@typescript-eslint/parser": "^5.30.5", as dependency on windows

**Review by prabhatsharma — APPROVED** [review]

**Review by hengfeiyang — COMMENTED** [review]
I reviewed @prabhatsharma,  i think we can merge it.

**Pull request merged**